sdk/cosmosdb/cosmos/package.json

```diff
@@ -94,6 +94,7 @@
     "@azure/core-auth": "^1.3.0",
     "@azure/core-rest-pipeline": "^1.2.0",
     "@azure/core-tracing": "^1.0.0",
+    "@azure/identity": "^2.0.1",
```

Comment:
Is there test gap? If so thoughts on test coverage?

Reply:
I will talk with the core team. this would be a good test to add.

Reply:
@jay-most any update?